SDPT-699 adjust the image resolution for the all posts page

vangork opened the pull request:
1. adjust the image resolution for the all posts page. change the height to 350px and scale down the image

2. change the apache and community hint from tooltip to popover

vangork committed:
1. adjust the image resolution for the all posts page. change the heith to 350px and scale down the image
2. change the apache and community hint from tooltip to popover

Signed-off-by: Luis Liu <luis_liu@dell.com>

twincitiesguy commented on assets/heliumjk/css/main.css:
```diff
@@ -1,4 +1,4 @@
 @import url("https://fonts.googleapis.com/css?family=Rubik:400,700|Montserrat:700");h1,h2,h3,h4,h5,h6{color:#444;font-family:"Montserrat",sans-serif;font-weight:700;letter-spacing:0px;margin-top:0}h1{font-size:36px}h2{font-size:30px}h3{font-size:26px}h4{font-size:22px}h5{font-size:18px}h6{font-size:16px}a{color:#EB586F;-moz-transition:.2s ease-in;-o-transition:.2s ease-in;-webkit-transition:.2s e ...
 @media (max-width: 480px){.navbar-brand img{max-width:120px}.container .navbar-toggler{padding:15px 15px!important;}}
-@media (max-width:767px){.container{margin-left:0; margin-right:0;width:100%;}.container .navbar-toggler{position: absolute;right:0px; padding:25px 15px; margin: 10px;}}.mb-25{margin-bottom:25px;}.div-catalog-img{overflow:hidden;max-height:150px;}.div-blog-img{overflow:hidden;max-height:400px;}.div-img{width:100%;height:auto}.text-version{color:#fff;background-color:#555;padding:1px 6px;border-ra ...
+@media (max-width:767px){.container{margin-left:0; margin-right:0;width:100%;}.container .navbar-toggler{position: absolute;right:0px; padding:25px 15px; margin: 10px;}}.mb-25{margin-bottom:25px;}.div-catalog-img{overflow:hidden;max-height:150px;}.div-blog-img{height:300px;display:flex;justify-content:center;align-items:center;}.div-img{width:auto;height:auto;max-width:100%;max-height:100%;displa ...
```
based on other input, seems like we may need to go even smaller on the images, or reformat the all-posts display

vangork replied:
Made another version. Please take a check.

twincitiesguy reviewed (changes requested):
Please also change the "blog" context path to "all-posts"

twincitiesguy reviewed (changes requested):
Please move the CSS changes out to its own file - it is too hard to see the changes in a compressed file

vangork commented:
> Please also change the "blog" context path to "all-posts"

done

vangork commented:
> Please move the CSS changes out to its own file - it is too hard to see the changes in a compressed file

done

vangork committed:
1. change all posts page link for blog/index.html to all-posts/index.html
2. recover the main.css to its original version and put all the customerization to site.css

Signed-off-by: Luis Liu <luis_liu@dell.com>

vangork committed:
1. change the schema of all post page
2. fix a bug on tag page

Signed-off-by: Luis Liu <luis_liu@dell.com>
